src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="category-edit">
+    <div class="category-edit__columns">
+      <section class="category-edit-editor">
+        <app-heading :level="1">カテゴリーの管理</app-heading>
+        <app-router-link
+          theme-color
+          underline
+          hover-opacity
+          :to="`/categories`"
```

Comment:
[nits]

変数展開しないのであれば、以下のような記述で良いと思います。

```suggestion
          to="/categories"
```

Reply:
[31c1e98](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/183/commits/31c1e9842bb5e050d9b4d0cc819969477c4465cf)

[55a8533](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/183/commits/55a8533e9012eb5ae6d42cb1142aa3f5dd15853c)